daprdocs/data/components/state_stores/generic.yaml

```diff
@@ -42,6 +42,17 @@
     etag: true
     ttl: false
     query: false
+- component: Coherence
+  link: setup-coherence
+  state: Alpha
+  version: v1
+  since: "1.15"
+  features:
+    crud: true
+    transactions: false
+    etag: false
+    ttl: true
```

Comment:
Given this has TTL enabled, do you want to provide any additional details on the operation of the TTL as a subheading for people to understand its usage patterns? Typically from any of the product docs.

Reply:
good suggestion.